packages/ag-charts-community/src/scene/gradient/gradient.ts

```diff
@@ -57,6 +57,11 @@ export abstract class Gradient {
             c0 = c1;
         }
 
+        if ('createPattern' in gradient) {
+            // Node-canvas stubs
+            gradient = (gradient as any).createPattern();
```

Comment:
Nit: Can we avoid the `as any` by specifying an explicit type for `gradient`?

Example:
```
let gradient: { createPattern?: ()=>unknown } = this.createCanvasGradient(ctx, bbox);
```

Reply:
Updated